doc/faq.md

```diff
@@ -28,6 +28,14 @@ That said, it is definitely possible to have write patterns that defeat this. De
 
 Theoretically, definitely. In Git, for example, the concept of "shallow clones" exists, and we could do something similar in Noms. This has not been implemented yet.
 
+### Is this another one of those NoSQL database thingies?
+
+It depends on the specific features and/or functionality you use to define `NoSQL`. Does Noms support similar type of data? Yes. Does Noms support a flexible schema? Yes.  Does Noms "compromise consistency"? No. Is Noms “eventually consistent”? No, it is always consistent.
```

Comment:
Agreed - it's kind of hard to answer which is why I would rather have it answered here and point to it than try to answer it in ad hoc fashion. Can we answer it in a similar way to what you just described above:

`Noms is not currently a distributed database with a multi-master architecture (e.g. Mongo). Hey wait, you said it was distributed - see clarification above. Hey wait, you said it was a database - see clarification above. Is it _anything_ like Mongo? Noms is an open-source database that supports data with a flexible schema, including queries and indices.` 

Or if you would prefer to leave it unanswered that's fine too. What about the Git question?